### ISSUE TITLE
Repository not current with demo.productionready.io/main.css

### ISSUE THREAD

**Issue by tecfu** [GoThinkster/conduit-bootstrap-template#5]
The stylesheet compiled from this repo does not match the CDN stylesheet that all of the demo apps are running: https://demo.productionready.io/main.css.

Can you please publish the correct source?

**Comment by amrit123456789** [comment]
yeah..having trouble on how to integrate this template with backend..someone please help!

**Comment by cirosantilli** [comment]
Yes... this might be the same as was mentioned at: https://github.com/gothinkster/conduit-bootstrap-template/issues/1 So broken.

https://demo.productionready.io/main.css looking correct:

![Screenshot from 2021-04-29 11-04-02](https://user-images.githubusercontent.com/1429315/116534642-b486c480-a8da-11eb-9a5a-a2323a1186a4.png)

This repo at 0b45af7bb43c2a038f09215885ff4cdb87b47a9b all broken:

![Screenshot from 2021-04-29 11-03-33](https://user-images.githubusercontent.com/1429315/116534643-b51f5b00-a8da-11eb-9690-0535732ca51d.png)

Tested with this fullstack integration: https://github.com/cirosantilli/node-express-sequelize-realworld-example-app/tree/46d32f773388700f35333152ce65f144064ebd82

cc. @EricSimons do you have the up-to-date source lying around in your machine by any chance? :smile_cat: :smile_cat: :smile_cat: :smile_cat:
